docs/aws/README.md

```diff
@@ -48,7 +48,12 @@ EOM
 
 ### Associate the IAM Role with your Kubernetes Service Account
 These commands will associate the AWS IAM Policy you created above with the Kubernetes Service Account used by Karpenter.
+The operation depends on the karpenter namespace, which will be created if missing.
 ```
+if ! kubectl get namespaces -o json | jq -r ".items[].metadata.name" | grep karpenter; then
```

Comment:
I'm a little wary of inline bash. WDYT about
```
cat << EOF | kubectl apply -f -
apiVersion: v1
kind: Namespace
metadata:
  name: karpenter
EOF
```

Reply:
Good suggestion, and to your point the bash statement depends on jq which is not a given to be present.